templates/og-member-count.html.twig

```diff
@@ -0,0 +1,22 @@
+{#
+/**
+ * @file
+ * Default theme implementation to show the member count of a group.
+ *
+ * Available variables:
+ * - count: The number of members in the group.
+ * - membership_states: An array of membership states included in the count.
+ * - group_label: The group label.
```

Comment:
The `group` variable is also available as we don't unset it in `template_preprocess_og_member_count()`. If we don't need this variable, we probably want to pass the label from the beginning, in the block plugin and remove the template preprocess function. Or we add here the `group` too.